cmd/clusterctl/cmd/completion.go

```diff
@@ -0,0 +1,233 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+const completionBoilerPlate = `
+# Copyright 2020 The Kubernetes Authors.
```

Comment:
Nit: For generated files, we can remove the year, see https://github.com/kubernetes-sigs/cluster-api/blob/master/hack/boilerplate/boilerplate.generatego.txt